tests/pkg/fixtures/webhook_queries.go

```diff
@@ -0,0 +1,87 @@
+package fixtures
+
+import (
+	"context"
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/tests/pkg/assertions"
+	"github.com/kyma-incubator/compass/tests/pkg/testctx"
+	gcli "github.com/machinebox/graphql"
+	"github.com/stretchr/testify/require"
+)
+
+func AddWebhookToRuntime(t require.TestingT, ctx context.Context, gqlClient *gcli.Client, in *graphql.WebhookInput, tenant, runtimeID string) *graphql.Webhook {
```

Comment:
More general question about all webhook fixtures - how did you decide where to provide tenant and where not? For example, all usages of `AddWebhookToApplication ` have `tenant` provided but here for `AddWebhookToRuntime` we do not provide any?

What does it mean to add a webhook to runtime with an empty tenant? Most probably the webhook will be "globally" but maybe it's important how we get these webhooks in the business logic - globally without tenant or with specified tenant?

Reply:
I looked at how they were created in their respective tests from which the examples come from. Usually it's the same way the resources are created (same client and tenant), but for the runtime I just referenced how it was done in the resynchronizing formaiton tests. I need to debug the test to see how it is created for runtime specifically, all other resources are straightforward.